### PR TITLE
[zh] add page: /zh/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/ fix 24086

### LINKED ISSUE #24086: [zh] add page: /zh/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity/
**This is a Feature Request**
page /tasks/configure-pod-container/assign-pods-nodes-using-node-affinity is not available from chinese version.

**What would you like to be added**
1. add file: content/zh/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md
2. translate the file from its english version

**Why is this needed**
Chinese and English version synchronization

**Comments**

### PR DESCRIPTION
**Pull request by zhiguo-lu**
1. add file: content/zh/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md
2. translate the file from its english version

close #24086

### CODE REVIEW

File: content/zh/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+    ```shell
+    kubectl get nodes --show-labels
+    ```
+    <!--
+    The output is similar to this:
+    -->
+    输出类似于此：
+
+    ```shell
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:47]: remove 'shell'

File: content/zh/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+    在前面的输出中，可以看到 `worker0` 节点有一个 `disktype=ssd` 标签。
+
+<!--
+## Schedule a Pod using required node affinity
+
+This manifest describes a Pod that has a `requiredDuringSchedulingIgnoredDuringExecution` node affinity,`disktype: ssd`. 
+This means that the pod will get scheduled only on a node that has a `disktype=ssd` label. 
+-->
+## 依据强制的节点亲和性调度 Pod  {#schedule-a-Pod-using-required-node-affinity}
+下面清单描述了一个 Pod，它有一个节点亲和性配置 `requiredDuringSchedulingIgnoredDuringExecution` ， `disktype=ssd` 。
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:100]: 注意空行（99-100 之间）和空格（100 行中标点符号之间的空格不需要）。

File: content/zh/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+
+
+## {{% heading "whatsnext" %}}
+
+<!--
+Learn more about
+[Node Affinity](/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity).
+-->
+学习更多的
```
Comment (tengqm) [inline content/zh/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:184]: 进一步了解

**Review by tengqm — COMMENTED** [review]
Overall looks good, just some nits.

File: content/zh/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md (hunk 1 above)
Comment (zhiguo-lu) [reply content/zh/docs/tasks/configure-pod-container/assign-pods-nodes-using-node-affinity.md:47]: 厉害，看了好一会才反应过来。

**Comment by zhiguo-lu** [comment]
tangqm, 
All done, Thanks

**Pull request merged**